### DISCUSSION TITLE
I am getting error while testing

### DISCUSSION THREAD

**Discussion by venky19666** [cloudscape-design/components#1744] [Q&A]
Please help me

react-scripts test -- --config=jest.config.js

  Invalid testPattern --config=jest.config.js|--watch|--config|{"roots":["<rootDir>\\src"],"collectCoverageFrom":["src\\**\\*.{js,jsx,ts,tsx}","!src\\**\\*.d.ts"],"setupFiles":["D:\\projects\\aws\\tap\\node_modules\\.pnpm\\react-app-polyfill@3.0.0\\node_modules\\react-app-polyfill\\jsdom.js"],"setupFilesAfterEnv":["<rootDir>\\src\\setupTests.ts"],"testMatch":["<rootDir>\\src\\**\\__tests__\\**\\*.{js,jsx,ts,tsx}","<rootDir>\\src\\**\\*.{spec,test}.{js,jsx,ts,tsx}"],"testEnvironment":"jsdom","transform":{"^.+\\.(js|jsx|mjs|cjs|ts|tsx)$":"D:\\projects\\aws\\tap\\node_modules\\.pnpm\\react-scripts@5.0.1_@babel+plugin-syntax-flow@7.22.5_@babel+plugin-transform-react-jsx@7.22.1_fojonpx2bszkwleihzx6fojaei\\node_modules\\react-scripts\\config\\jest\\babelTransform.js","^.+\\.css$":"D:\\projects\\aws\\tap\\node_modules\\.pnpm\\react-scripts@5.0.1_@babel+plugin-syntax-flow@7.22.5_@babel+plugin-transform-react-jsx@7.22.1_fojonpx2bszkwleihzx6fojaei\\node_modules\\react-scripts\\config\\jest\\cssTransform.js","^(?!.*\\.(js|jsx|mjs|cjs|ts|tsx|css|json)$)":"D:\\projects\\aws\\tap\\node_modules\\.pnpm\\react-scripts@5.0.1_@babel+plugin-syntax-flow@7.22.5_@babel+plugin-transform-react-jsx@7.22.1_fojonpx2bszkwleihzx6fojaei\\node_modules\\react-scripts\\config\\jest\\fileTransform.js"},"transformIgnorePatterns":["[\\\\\\]node_modules[\\\\\\].+\\.(js|jsx|mjs|cjs|ts|tsx)$","^.+\\.module\\.(css|sass|scss)$"],"modulePaths":["D:\\projects\\aws\\tap\\src"],"moduleNameMapper":{"^react-native$":"react-native-web","^.+\\.module\\.(css|sass|scss)$":"identity-obj-proxy"},"moduleFileExtensions":["web.js","js","web.ts","ts","web.tsx","tsx","json","web.jsx","jsx","node"],"watchPlugins":["jest-watch-typeahead\\filename","jest-watch-typeahead\\testname"],"resetMocks":true,"rootDir":"D:\\projects\\aws\\tap"}|--env|D:\projects\aws\tap\node_modules\.pnpm\node_modules\jest-environment-jsdom\build\index.js supplied. Running all tests instead.
  Invalid testPattern --config=jest.config.js|--watch|--config|{"roots":["<rootDir>\\src"],"collectCoverageFrom":["src\\**\\*.{js,jsx,ts,tsx}","!src\\**\\*.d.ts"],"setupFiles":["D:\\projects\\aws\\tap\\node_modules\\.pnpm\\react-app-polyfill@3.0.0\\node_modules\\react-app-polyfill\\jsdom.js"],"setupFilesAfterEnv":["<rootDir>\\src\\setupTests.ts"],"testMatch":["<rootDir>\\src\\**\\__tests__\\**\\*.{js,jsx,ts,tsx}","<rootDir>\\src\\**\\*.{spec,test}.{js,jsx,ts,tsx}"],"testEnvironment":"jsdom","transform":{"^.+\\.(js|jsx|mjs|cjs|ts|tsx)$":"D:\\projects\\aws\\tap\\node_modules\\.pnpm\\react-scripts@5.0.1_@babel+plugin-syntax-flow@7.22.5_@babel+plugin-transform-react-jsx@7.22.1_fojonpx2bszkwleihzx6fojaei\\node_modules\\react-scripts\\config\\jest\\babelTransform.js","^.+\\.css$":"D:\\projects\\aws\\tap\\node_modules\\.pnpm\\react-scripts@5.0.1_@babel+plugin-syntax-flow@7.22.5_@babel+plugin-transform-react-jsx@7.22.1_fojonpx2bszkwleihzx6fojaei\\node_modules\\react-scripts\\config\\jest\\cssTransform.js","^(?!.*\\.(js|jsx|mjs|cjs|ts|tsx|css|json)$)":"D:\\projects\\aws\\tap\\node_modules\\.pnpm\\react-scripts@5.0.1_@babel+plugin-syntax-flow@7.22.5_@babel+plugin-transform-react-jsx@7.22.1_fojonpx2bszkwleihzx6fojaei\\node_modules\\react-scripts\\config\\jest\\fileTransform.js"},"transformIgnorePatterns":["[\\\\\\]node_modules[\\\\\\].+\\.(js|jsx|mjs|cjs|ts|tsx)$","^.+\\.module\\.(css|sass|scss)$"],"modulePaths":["D:\\projects\\aws\\tap\\src"],"moduleNameMapper":{"^react-native$":"react-native-web","^.+\\.module\\.(css|sass|scss)$":"identity-obj-proxy"},"moduleFileExtensions":["web.js","js","web.ts","ts","web.tsx","tsx","json","web.jsx","jsx","node"],"watchPlugins":["jest-watch-typeahead\\filename","jest-watch-typeahead\\testname"],"resetMocks":true,"rootDir":"D:\\projects\\aws\\tap"}|--env|D:\projects\aws\tap\node_modules\.pnpm\node_modules\jest-environment-jsdom\build\index.js supplied. Running all tests instead.
 FAIL  src/pages/tasks/tasks.spec.tsx
  ● Test suite failed to run
                                                                                                                   
    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation

    Details:

    D:\projects\aws\tap\src\pages\tasks\tasks.spec.tsx:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){import { render, screen } from "@testing-library/react";
                                                                                      ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      at Runtime.createScriptFromCode (node_modules/.pnpm/jest-runtime@27.5.1/node_modules/jest-runtime/build/index.js:1728:14

**Comment by YueyingLu** [comment]
Hi @venky19666 ,
thanks for reaching out to Cloudscape.
The issue you are having seems to be [jest](https://github.com/jestjs/jest) related. It could be the [transform](https://jestjs.io/docs/configuration#transform-objectstring-pathtotransformer--pathtotransformer-object) configuration. 
Please let us know if you have further issues with Cloudscape.
